manuscript/chapter3-contributor-mode.md

```diff
@@ -207,31 +258,32 @@ Let's move to the next item…
 
 #### Provide useful resources for newcomers
 
-Newcomers always have the same issue as we had ourselves in the past: they are disoriented. Transforming them in autonomous user is critical to lower their defenses, it's the reason Travellers' Club is famous, it lets travellers collect books about the cities they visited, to visit them as they wish.  
-In an Open Source project is not always possible to do what one wants, having a set of instructions is very helpful. It's like the concierge in a hotel explaining what to visit, where to eatand soon. It's nice and one feels welcomed.
+Newcomers always have the same issue as we had ourselves in the past: they are disoriented. Transforming them in autonomous users is critical to lower their defenses, it's the reason Travellers' Club is famous, it lets travellers collect books about the cities they visited, to visit them as they wish.  
+In an Open Source project is not always possible to do what one wants, having a set of instructions is very helpful. It's like the concierge in a hotel explaining what to visit, where to eat and soon. It's nice and one feels welcome.
 
 One of the suggestions or activities I think are among the most powerful to boost engagement in newcomers is proposing them to read the documentation because:
 
 * They discover more about the project, they get trained without losing time on mentoring/supporting them
 * They review the resources and may be able to signal if they need updating/corrections, if something is missing and if they are overall clear and understandable
 * Tomorrow they will know how they can use documentation and they will help in empowering it in the long run
 
-Reading is accessible to everyone and doesn't require a lot of efforts or time (not to mention by reading one gains experience on the project) but it's a first important step to engage and involve the reader in being a part of it and to contribute back. Maybe fixing typos or updating screenshots included in the page.
+Reading is accessible to everyone and doesn't require a lot of efforts or time (not to mention by reading one gains experience on the project) but it's a first important step to engage and involve the reader in being a part of it and to contribute back. Maybe fixing typos or updating screenshots included in the page.  
+Read the documentation let the volunteers to discover how the project/community is structured, organized and what are the various web portal or tools that are important. As example in Mozilla, Pontoon is the web tool for localization instead in WordPress is GlotPress or in OpenSuse is Weblate. 
```

Comment:
Is Pontoon still in use?
If I recall correctly, there was a change, back when activate.mozilla.community was still in use …

Reply:
yes it is still used